modules/spring-config/src/main/java/org/mule/runtime/config/spring/internal/MuleDocumentLoader.java

```diff
@@ -58,7 +60,48 @@ public void handle(short operation, String key, Object data, Node src, Node dst)
     }
   };
 
-  private final DocumentLoader defaultLoader = new DefaultDocumentLoader();
+  /**
+   * JAXP attribute used to configure the schema language for validation.
+   */
+  private static final String SCHEMA_LANGUAGE_ATTRIBUTE = "http://java.sun.com/xml/jaxp/properties/schemaLanguage";
+
+  /**
+   * JAXP attribute value indicating the XSD schema language.
+   */
+  private static final String XSD_SCHEMA_LANGUAGE = "http://www.w3.org/2001/XMLSchema";
+
+  private final DocumentLoader defaultLoader = new DefaultDocumentLoader() {
```

Comment:
It doesn't allow passing a factory as argument, that's what I'm changing (instead of `DefaultBuilderFactory.newInstance()` it uses `XMLSecureFactories.createDefault().getDocumentBuilderFactory()`). This is because including xerces in the classpath changes its behavior, as with many other libraries.